apps/hash-graph/docker/Dockerfile

```diff
@@ -5,22 +5,29 @@ ENV RUSTUP_HOME=/usr/local/rustup \
     PATH=:$PATH:/usr/local/cargo/bin
 
 COPY rust-toolchain.toml .
-RUN apk add --no-cache gcc musl-dev && \
+RUN apk add --no-cache gcc musl-dev bash && \
     wget -q -O- https://sh.rustup.rs | sh -s -- -y --default-toolchain none --profile minimal && \
     rustup show
 
+SHELL ["bash", "-c"]
+
 FROM rust AS builder
 WORKDIR /usr/local/src/hash-graph
 COPY . .
 ARG PROFILE=production
+ARG TYPE_FETCHER=yes
 
 # To be removed once https://github.com/open-telemetry/opentelemetry-rust/issues/934 is sorted
-RUN apk add --no-cache make openssl-dev protobuf-dev
+RUN apk add --no-cache make protobuf-dev
 
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/usr/local/cargo/git \
     --mount=type=cache,target=/usr/local/src/hash-graph/target \
-    cargo install --path bin/cli --profile $PROFILE --locked
+    if [[ ${TYPE_FETCHER^^} == Y* || ${TYPE_FETCHER^^} == T* || $TYPE_FETCHER == 1 ]] ; then  \
```

Comment:
I tried to capture most common boolean flags like `yes`, `YES`, `true`, `True`, and `1`

Reply:
I haven't tested it, but even `YesPleaseFFS!` should work 😆